src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -26,6 +26,7 @@ import spray.routing.RequestContext
 import scala.collection.JavaConversions._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Try}
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
rob title is recommending this instead of using a global:
implicit val ec: ExecutionContext = system.dispatcher